apps/flame-defi/app/pool/modules/pool-position-details/sections/content-section.tsx

```diff
@@ -5,26 +5,34 @@ import { usePoolPositionContext, usePoolContext } from "pool/hooks";
 import { PoolTxnSteps, PriceRangeBlock, TokenInfoCard } from "pool/components";
 import { ConfirmationModal } from "components/confirmation-modal/confirmation-modal";
 import { Skeleton, Switch } from "@repo/ui/components";
+import { TXN_STATUS } from "@repo/flame-types";
 
 export const ContentSection = () => {
-  const { modalOpen, setModalOpen, txnStatus } = usePoolContext();
+  const { modalOpen, setModalOpen } = usePoolContext();
   const {
     selectedSymbol,
     handleReverseTokenData,
     poolToken0,
     poolToken1,
-    collectAsNative,
-    handleCollectAsNative,
+    isCollectAsWrappedNative,
+    handleIsCollectAsWrappedNative,
     isReversedPoolTokens,
   } = usePoolPositionContext();
   const poolTokens = isReversedPoolTokens
     ? [poolToken1, poolToken0]
     : [poolToken0, poolToken1];
   const token0 = poolTokens[0] || null;
   const token1 = poolTokens[1] || null;
+  const hasUnclaimedFees =
+    token0?.unclaimedFees &&
+    token1?.unclaimedFees &&
+    token0.unclaimedFees > 0 &&
+    token1.unclaimedFees > 0
+      ? true
```

Comment:
When you have a ternary that returns true or false on the right hand side, you probably shouldn't have a ternary.
If you wrap the first two expressions in `Boolean()`, then the whole expression should return a boolean, because the return type of `>` expression is already boolean.

Reply:
You have to wrap it in a Boolean or have the ternary return true or false or the value is hasUnclaimedFees is actually set to 0. I can use Boolean if you don't like the ternary.